src/main/resources/db/changelog/changes/create-audit-table.xml

```diff
@@ -23,4 +23,11 @@
     </createTable>
   </changeSet>
 
+  <changeSet id="MODDCB-75" author="dbykov">
```

Comment:
I believe every change related to the audit table should be in a single file.
@Vignesh-kalyanasundaram suggest, please

Reply:
Since this code is already deployed in integration environment, I feel adding the changeset into an existing xml will break something. Please test it in rancher before merge it. Thanks

Reply:
tested. it works completely properly. Thanks